docs/team/e0958902.md

```diff
@@ -20,4 +21,5 @@ The following contains my contribution summary and pull request reviews.
 
 ### PR Reviews
 #9, #18, #20, #23, #53, #73, #75, #84, #88, 
-#89, #90, #95, #98, #115, #116
\ No newline at end of file
+#89, #90, #95, #98, #115, #116, #126, #127, 
+#129, #134, #139, #
```

Comment:
Did you mean to add more PR numbers here? Else, it might be better to clean up any trailing `#`

Reply:
Resolved in [bbba1d1](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/206/commits/bbba1d1d4f37f0a01771a14fc1a6c72f4c0aaa9d)